tests/spec/code_series_ops/test_containment.py

```diff
@@ -27,3 +27,27 @@ def test_is_in(spec_test):
             4: None,
         },
     )
+
+
+def test_is_in_codelist_csv(spec_test):
+
+    codelist = codelist_from_csv_lines(
+        [
+            "code",
+            "abc",
+            "ghi",
+        ],
+        column="code",
+        system="snomedct",
+    )
+
+    spec_test(
+        table_data,
+        p.c1.is_in(codelist),
```

Comment:
Might need to do some work to get this to render sensibly in docs.  An escape hatch might be best here?

Reply:
That sounds sensible. I'm not sure what the means concretely though. Is it something I need to think about now?